sdk/core/core-http/src/util/utils.ts

```diff
@@ -154,14 +154,21 @@ export function promiseToCallback(promise: Promise<any>): Function {
     throw new Error("The provided input is not a Promise.");
   }
   return (cb: Function): void => {
```

Comment:
I wonder if anything even uses this function...

Reply:
I've made this issue to continue with the effort of cleaning up unused code in core-http: https://github.com/Azure/azure-sdk-for-js/issues/8973

@ramya-rao-a I've assigned #8972 to myself, but in case you know a better resource for that issue, feel free to re-assign it.